core/src/components/select-popover/select-popover.md.scss

```diff
@@ -2,7 +2,11 @@
 @import "./select-popover.md.vars";
 
 ion-list ion-radio::part(container) {
-  opacity: 0;
```

Comment:
I could not find a compelling reason for the use of `opacity: 0` here.

The changes comes from: https://github.com/ionic-team/ionic-framework/pull/23474 and I believe is targeted around this issue: https://github.com/ionic-team/ionic-framework/issues/12310 (the PR closes 3 issues).

Reply:
Looks like this fixes another issue we never caught where the label text did not extend the full width of the row (because the radio container was still taking up space)

| Before | After |
| - | - | 
| ![Screenshot 2024-04-29 at 10 16 16 AM](https://github.com/ionic-team/ionic-framework/assets/2721089/bbe16c6a-a0cd-45fe-9e46-60113ff2ce91) | ![Screenshot 2024-04-29 at 10 16 21 AM](https://github.com/ionic-team/ionic-framework/assets/2721089/47e9f77e-8f29-490b-91fd-09cc669240d5) |